homeassistant/components/whirlpool/strings.json

```diff
@@ -74,6 +74,37 @@
           "door_open": "Door Open"
         }
       },
+      "dryer_state": {
+        "name": "State",
```

Comment:
this can also be a reference. You can also consider just setting the name of the entity to `None` as this kinda is the main feature of the device